contributed_definitions/nyaml/NXdispersive_material.yaml

```diff
@@ -51,6 +63,7 @@ NXdispersive_material(NXobject):
         Denotes whether the dispersion is calculated or derived from an experiment
       enumeration: [measured, simulated]
     REFERENCES(NXcite):
+      nameType: partial
```

Comment:
nameType: any cuz if it were partial also empty string would be possible as their is no lower case letter